lib/inspec/resources/oracledb_session.rb

```diff
@@ -48,18 +46,16 @@ def query(sql)
       if @sqlcl_bin && inspec.command(@sqlcl_bin).exist?
         @bin = @sqlcl_bin
         format_options = "set sqlformat csv\nSET FEEDBACK OFF"
-        parser = :parse_csv_result
       else
         @bin = "#{@sqlplus_bin} -S"
-        format_options = "SET MARKUP HTML ON\nSET PAGESIZE 32000\nSET FEEDBACK OFF"
-        parser = :parse_html_result
+        format_options = "SET MARKUP CSV ON\nSET PAGESIZE 32000\nSET FEEDBACK OFF"
       end
 
       command = command_builder(format_options, sql)
       inspec_cmd = inspec.command(command)
 
       DatabaseHelper::SQLQueryResult.new(inspec_cmd,
-                                         send(parser, inspec_cmd.stdout))
+                                         send(:parse_csv_result, inspec_cmd.stdout))
```

Comment:
not on you... but why is this a `send` at all?

Reply:
Wasn't sure on the reasoning behind this either. Guess it could be updated to this instead...
```
DatabaseHelper::SQLQueryResult.new(inspec_cmd, parse_csv_result(inspec_cmd.stdout))
```